nano/nano_wallet/entry.cpp

```diff
@@ -256,9 +256,11 @@ int main (int argc, char * const * argv)
 {
 	nano::set_umask ();
 	nano::node_singleton_memory_pool_purge_guard memory_pool_cleanup_guard;
+
+	QApplication application (argc, const_cast<char **> (argv));
```

Comment:
I tried making this change only and it seemed to address the issue of the dialog not popping up correctly.

Specifically it didn't require making the substitution: application./QApplication::

Reply:
Ah, the `QApplication`/`application.` is unrelated, that was a clang-tidy warning telling us that we're invoking a static function via an object uselessly, the actual reason for which the crash happened was instantiating `QMessageBox` without a `QApplication` instance.